### PR TITLE
[NUI] Remove wrong force casting, Layer as View

### PR DESCRIPTION
**Pull request by hinohie**
Remove force casting at LayoutItem.cs
just change `Owner.GetParent() as View;`
+
tiny typo error fix

### CODE REVIEW

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs [hunk 1]
```diff
@@ -168,7 +168,10 @@ internal void AttachToOwner(View owner)
             OnAttachedToOwner();
             // Add layout to parent layout if a layout container
             View parent = Owner.GetParent() as View;
-            (parent?.Layout as LayoutGroup)?.Add( this );
+            if (parent != null)
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/Layouting/LayoutItem.cs:171]: Do we need this change?

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs [hunk 2]
```diff
@@ -551,8 +554,7 @@ private bool SetFrame(float left, float top, float right, float bottom)
                                                          " top:" + _layoutPositionData.Top +
                                                          " right:" + _layoutPositionData.Right +
                                                          " bottom:" + _layoutPositionData.Bottom );
-                Container onwerContainer = Owner.GetParent();
-                View onwerView = onwerContainer is Layer ? new View(Layer.getCPtr(onwerContainer).Handle, false) : onwerContainer as View;
+                View onwerView = Owner.GetParent() as View;
```
Comment (bshsqa) [inline src/Tizen.NUI/src/public/Layouting/LayoutItem.cs:557]: Layer is not a View.
If Parent is Layer it is not safe to cast directly from Layer to View.

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs (hunk 1 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/Layouting/LayoutItem.cs]: Nope. Revert.

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs (hunk 2 above)
Comment (hinohie) [reply src/Tizen.NUI/src/public/Layouting/LayoutItem.cs]: As I know, `as` operation is safe enough when casting failed. If it is not safe,.. We have to fix a lot of codes.

Comment (huiyueun) [reply src/Tizen.NUI/src/public/Layouting/LayoutItem.cs]: I think it is safe code. If the parent is Layer, there is no additional work required.

**Review by huiyueun — APPROVED** [review]

**Pull request merged**